scripts/g.extension/g.extension.py

```diff
@@ -207,14 +208,210 @@
     MAKE = "make"
 
 
+class GitAdapter:
+    """
+    Basic class for listing and downloading GRASS GIS AddOns using git
+
+    """
+
+    def __init__(
+        self,
+        url="https://github.com/osgeo/grass-addons",
+        working_directory=None,
+        official_repository_structure=True,
+        major_grass_version=8,
```

Comment:
Another place which has the hardcoded major version. See all the desperate commits trying to fix version numbers around every release.

Reply:
Thanks! Yes, that was planned to be set to version extracted at the bottom of the module, forgot to plug that in...